### PR TITLE
Made annotation attributes remove button display on hover

### PR DESCRIPTION
**Pull request by jeradrutnam**

### CODE REVIEW

File: modules/web/css/ballerina-file-editor.css [hunk 1]
```diff
@@ -892,7 +892,7 @@ input#package-name-input {
 }
 
 .annotation-table td{
-    height: 20px;
+    height: 18px;
```
Comment (hemikak) [inline modules/web/css/ballerina-file-editor.css]: Having height 18px might cause an issue with calculating the height of the container as the default height is taken as 20px. Have to update here as well https://github.com/ballerinalang/composer/blob/master/modules/web/js/ballerina/visitors/sizing-utils.js#L595 .

Comment (jeradrutnam) [inline modules/web/css/ballerina-file-editor.css]: Shall we change it there as well? And I tested after changing it, but didn't overcome any issue.

Comment (jeradrutnam) [inline modules/web/css/ballerina-file-editor.css]: Reverted this and made a different change to overcome this

**Comment by jeradrutnam** [comment]
Changes moved to #1936

**Pull request closed without merging**